### ISSUE TITLE
On 4.6.1-1 tag temporary password is not shown on console when container started

### ISSUE THREAD

**Issue by raedkit** [qbittorrent/docker-qbittorrent-nox#19]
Hi, I want to report the following issue :
When starting the container and no WebUI passwords exists or default password (adminadmin), now qBittorrent (v4.6.1) generate a random password and show it on the console.
The issue, is that the password is not showing on the console and only shows when terminating the container.
Could you please solve this issue ? Thanks in advance.

Example : 
- Stopped container
```
2023-11-21 16:57:44.767	chown: /downloads: No such file or directory
2023-11-21 16:57:44.919	WebUI will be started shortly after internal preparations. Please wait...
2023-11-21 17:02:28.101	Catching signal: SIGTERM
2023-11-21 17:02:28.101	Exiting cleanly
2023-11-21 17:05:19.252	******** Information ********
2023-11-21 17:05:19.252	To control qBittorrent, access the WebUI at: http://localhost:20000
2023-11-21 17:05:19.252	The WebUI administrator username is: admin
2023-11-21 17:05:19.252	The WebUI administrator password was not set. A temporary password is provided for this session: fs9USZcag
2023-11-21 17:05:19.252	You should set your own password in program preferences.
```

As you could see, the password is logged after SIGTERM and exiting cleanly

Below, you could find an example of a started container : 
```
2023-11-21 17:05:47.316	chown: /downloads: No such file or directory
2023-11-21 17:05:47.459	WebUI will be started shortly after internal preparations. Please wait...
```
As you could see, the information is not logged until receiving SIGTERM

**Comment by Chocobo1** [comment]
Please follow https://github.com/qbittorrent/qBittorrent/issues/19984

**Comment by Chocobo1** [comment]
Fixed in 4.6.2.
